core/templatetags/is_saved_by.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+from django.contrib.auth import get_user_model
+
+from core.models import Post
+
+register = template.Library()
+
+User = get_user_model()
+
+
+@register.filter
+def is_saved_by(post: Post, user: User) -> bool:

```

Comment:
Similar here, this name is misleading..
Maybe only "is_saved"? or is_post_saved?

Reply:
Naming is one of the most difficult things. Thanks